zgrab_schema.py

```diff
@@ -202,7 +202,8 @@
     }),
     "session_ticket":SubRecord({
         "value":Binary(),
-        "length":Integer()
+        "length":Integer(),
```

Comment:
What's the point of having length? Length is implicitly determined by the base64 value stored in the `value` field in JSON.